ci/scripts/python_wheel_manylinux_build.sh

```diff
@@ -64,6 +64,7 @@ echo "=== (${PYTHON_VERSION}) Building Arrow C++ libraries ==="
 : ${ARROW_WITH_LZ4:=ON}
 : ${ARROW_WITH_SNAPPY:=ON}
 : ${ARROW_WITH_ZLIB:=ON}
+: ${ARROW_WITH_UTF8PROC:=ON}
```

Comment:
Are these necessary?

Reply:
No, in the CmakeOptions.cmake we set the variable if the Gandiva or Kernel is computed.